build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
@devlead what is the server 32 bit? Can't we change that?  Reluctant to change the build script specifically for when running on our Jenkins server.